emri_sur_load.py

```diff
@@ -119,32 +119,57 @@ def alpha_scaling_time(q):
     t_scaled=np.array(time)*alpha
     return t_scaled
 
-def slog_surrogate(q, h_eim_amp_spline, h_eim_ph_spline, eim_indicies_amp, eim_indicies_ph, B_amp, B_ph):
+def slog_surrogate(q, h_eim_amp_spline, h_eim_ph_spline, eim_indicies_amp, eim_indicies_ph, B_amp, B_ph, calibrated):
     """ Compute the interpolated waveform for a single mode """
     h_eim_amp = np.array([splev(np.log(q), h_eim_amp_spline[j])  for j in range(len(eim_indicies_amp))])
     h_eim_ph = np.array([splev(np.log(q), h_eim_ph_spline[j]) for j in range(len(eim_indicies_ph))])
     h_approx_amp = np.dot(B_amp.transpose(), h_eim_amp)
     h_approx_ph = np.dot(B_ph.transpose(), h_eim_ph)
     h_approx = amp_ph_to_comp(h_approx_amp, h_approx_ph)
-    h_approx = alpha_scaling_h(q,h_approx)*(1/q) # because the training waveform follows definition q<1 and we follow q>1
-    return h_approx
+    if calibrated==True:
+        h_approx = alpha_scaling_h(q,h_approx) # because the training waveform follows definition q<1 and we follow q>1
```

Comment:
this code comment should be associated with the line below "return np.array(h_approx)*(1/q)"